src/fromager/sdist.py

```diff
@@ -195,7 +195,16 @@ def handle_requirement(
                 | build_sdist_dependencies,
             )
             try:
-                sources.build_sdist(ctx, req, sdist_root_dir)
+                find_sdist_result = finders.find_sdist(
+                    ctx.sdists_builds, req, resolved_version
+                )
+                if not find_sdist_result:
+                    logger.info(f"{req.name}: already resolved sdist version not found")
+                    sources.build_sdist(ctx, req, sdist_root_dir)
+                else:
+                    logger.info(
+                        f"{req.name} has sdist version {resolved_version}: {find_sdist_result}"
```

Comment:
```suggestion
                        f"{req.name}: have sdist version {resolved_version}: {find_sdist_result}"
```

Minor tweak to the format of the message for consistency.